specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -172,8 +172,15 @@ interface TextGroundednessDetectionOperations {
   @doc("A synchronous API for the analysis of language model outputs to determine alignment with user-provided information or identify fictional content.")
   @route("/text:detectGroundedness")
   @post
-  detectGroundedness is Azure.Core.RpcOperation<
-    DetectGroundednessOptions,
-    DetectGroundednessResult
+  detectGroundednessOptions is Azure.Core.RpcOperation<
+    {
+      @body
+      @doc("The text groundedness detection request.")
+      @projectedName("csharp", "options")
+      @projectedName("python", "options")
+      @projectedName("java", "options")
```

Comment:
@johanste or @tjprescott, is there a generic way to do this for all languages we may use in the future? Or is it better just to do a client tsp?

Reply:
I‘ve updated this to clientName